### PR TITLE
Feature/updated animals table

### PR DESCRIPTION
**Pull request by ShinhyoBelok**
# Changes 🔨 

* Add 7 animals to the animals' table.
* Use transactions, savepoint, and rollback.
* Add queries to answer the logical questions.

## Screenshots.

Captura de pantalla 2023-02-22 092011
Captura de pantalla 2023-02-22 092116
Captura de pantalla 2023-02-22 092440
Captura de pantalla 2023-02-22 091845
Captura de pantalla 2023-02-21 211804
Captura de pantalla 2023-02-21 212158
Captura de pantalla 2023-02-21 212407
Captura de pantalla 2023-02-21 214256
Captura de pantalla 2023-02-21 211408
Captura de pantalla 2023-02-21 211650

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -56,4 +56,109 @@ VALUES (
...
   true,
   11
+);
+
+INSERT INTO animals (
+  name,
+  date_of_birth,
+  escape_attempts,
+  neutered,
+  weight_kg
```
Comment (Davidosky007) [inline data.sql:66]: ## Kindly ensure that the right weight values are imputed as you can see some have negative values

File: queries.sql [hunk 2]
```diff
@@ -5,4 +5,28 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+BEGIN;
+UPDATE animals SET species = 'unspecified';
```
Comment (Davidosky007) [inline queries.sql:10]: * Kindly verify that the changes were made here

File: queries.sql [hunk 3]
```diff
@@ -5,4 +5,28 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+ROLLBACK;
```
Comment (Davidosky007) [inline queries.sql:12]: * Kindly verify that the changes were undone here

File: queries.sql [hunk 4]
```diff
@@ -5,4 +5,28 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
```
Comment (Davidosky007) [inline queries.sql:16]: * Kindly verify that the changes were made here

File: queries.sql [hunk 5]
```diff
@@ -5,4 +5,28 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+COMMIT;
```
Comment (Davidosky007) [inline queries.sql:18]: * Kindly verify that the changes persist after commit

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* No linter errors 
* Well written PR description 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by addod19 — APPROVED** [review]
### Hi @ShinhyoBelok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job fixing the values and also making the verifications :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**